### PR TITLE
docs: simplify docs

### PR DESCRIPTION
**Pull request by UziTech**
- remove unused js and css
- update repo information
- simplify build-docs script

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -77,4 +77,4 @@ xcuserdata
 *.env
 
 # built docs
-public
+/public
```
Comment (jsumners) [inline .gitignore:80]: Should it be? 
```suggestion
public/
```

`/public` makes me think it is going to try to ignore a "public" directory at the root of the file system, not the project.

Comment (UziTech) [reply .gitignore:80]: No `public/` will make it ignore any folder named `public` in the project. `/public` is the only way to make it only ignore a folder named `public` in the root of the project.

Comment (jsumners) [reply .gitignore:80]: 👍

**Comment by jsumners** [comment]
⚠️ This issue has been locked due to age. If you have encountered a recent
problem that seems to be covered by this issue, please open a new issue.

Please include a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example)
when opening a new issue.

**Pull request merged**